samples/java_layout/src/main/java/io/customer/android/sample/java_layout/sdk/CustomerIORepository.java

```diff
@@ -51,64 +58,40 @@ public void initializeSdk(SampleApplication application) {
 
         // Finally, build to finish initializing the SDK.
         builder.build();
-
-        // TODO: Remove old builder and use new builder to initialize the SDK
-        // New method to initialize CustomerIO
-        // The new method should be called after the old method till the old method is removed
-        // This is because the push and in-app modules are still using properties only initialized in the old method
-        CustomerIOBuilder newBuilder = new CustomerIOBuilder(application, sdkConfig.getCdpApiKey());
-
-        newBuilder.setMigrationSiteId(sdkConfig.getSiteId());
-        if (sdkConfig.getBackgroundQueueMinNumOfTasks() != null) {
-            newBuilder.setFlushAt(sdkConfig.getBackgroundQueueMinNumOfTasks());
-        }
-        if (sdkConfig.getBackgroundQueueSecondsDelay() != null) {
-            newBuilder.setFlushInterval(sdkConfig.getBackgroundQueueSecondsDelay().intValue());
-        }
-        newBuilder.setAutoTrackDeviceAttributes(Boolean.TRUE.equals(sdkConfig.isDeviceAttributesTrackingEnabled()));
-        newBuilder.setAutoTrackActivityScreens(Boolean.TRUE.equals(sdkConfig.isScreenTrackingEnabled()));
-
-        // Enable detailed logging for debug builds.
-        if (sdkConfig.debugModeEnabled()) {
-            newBuilder.setLogLevel(CioLogLevel.DEBUG);
-        }
-
-        builder.addCustomerIOModule(new ModuleMessagingPushFCM());
-        builder.addCustomerIOModule(new ModuleMessagingInApp(
-                new MessagingInAppModuleConfig.Builder()
-                        .setEventListener(new InAppMessageEventListener(appGraph.getLogger()))
-                        .build()
-        ));
-        newBuilder.build();
     }
 
     /**
      * Makes modifications to SDK configurations. It is mainly for testing
      * purposes and may not be needed unless there is a need to override any
      * default configuration from the SDK.
      */
-    private void configureSdk(CustomerIO.Builder builder, final CustomerIOSDKConfig sdkConfig) {
-        final String trackingApiUrl = sdkConfig.getTrackingURL();
-        if (!TextUtils.isEmpty(trackingApiUrl)) {
-            builder.setTrackingApiURL(trackingApiUrl);
+    private void configureSdk(CustomerIOBuilder builder, final CustomerIOSDKConfig sdkConfig) {
+        builder.setMigrationSiteId(sdkConfig.getSiteId());
+
+        final String apiHost = sdkConfig.getApiHost();
+        if (!TextUtils.isEmpty(apiHost)) {
+            builder.setApiHost(apiHost);
+        }
+
+        final String cdnHost = sdkConfig.getCdnHost();
+        if (!TextUtils.isEmpty(cdnHost)) {
+            builder.setCdnHost(cdnHost);
         }
```

Comment:
Same as https://github.com/customerio/customerio-android/pull/337#discussion_r1603966585

Reply:
Replied in linked comment, we only allow non-null values here today.